pkg/services/ngalert/notifier/channels/pagerduty.go

```diff
@@ -23,13 +23,14 @@ const (
 	pagerDutyEventTrigger = "trigger"
 	pagerDutyEventResolve = "resolve"
 
-	defaultSeverity = "critical"
+	defaultSeverity = "error"
```

Comment:
I wonder if we should submit this line's change in a separate PR, with `add to changelog`? That way, this change gets called out specifically in the patch notes.

Reply:
I think we will do this later as part of unification process.